### PR TITLE
feat: run tests against live testnets

### PR DESCRIPTION
**Pull request by deanamiel**
PR to allow running uint tests against live testnets.

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
 require('@nomicfoundation/hardhat-toolbox');
 require('solidity-coverage');
 
+const fs = require('fs');
+const env = process.env.ENV || 'testnet';
+const {
+  importNetworks,
+} = require('@axelar-network/axelar-contract-deployments/evm/utils');
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const keys = fs.existsSync(`${__dirname}/info/keys.json`)
```
Comment (milapsheth) [inline hardhat.config.js:10]: can make it a bit cleaner
https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/hardhat.config.js#L3

Comment (deanamiel) [reply hardhat.config.js]: fixed

File: test/Operators.js [hunk 2]
```diff
@@ -69,29 +75,40 @@ describe('Operators', () => {
...
-        operators.connect(userWallet).removeOperator(operatorWallet.address),
+        operators
+          .connect(userWallet)
+          .removeOperator(operatorWallet.address, getGasOptions()),
       ).to.be.revertedWithCustomError(operators, 'NotOwner');
     });
 
     it('should remove operator', async () => {
-      await operators.connect(ownerWallet).addOperator(operatorWallet.address);
+      console.log(JSON.stringify(getGasOptions()));
```
Comment (milapsheth) [inline test/Operators.js:103]:
```suggestion
```

**Review by milapsheth — APPROVED** [review]

File: hardhat.config.js [hunk 3]
```diff
@@ -1,6 +1,15 @@
 require('@nomicfoundation/hardhat-toolbox');
 require('solidity-coverage');
 
+const env = process.env.ENV || 'testnet';
+const {
+  importNetworks,
+  readJSON,
+} = require('@axelar-network/axelar-contract-deployments/evm/utils');
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const keys = readJSON(`${__dirname}/info/keys.json`);
```
Comment (milapsheth) [inline hardhat.config.js:10]:
```suggestion
const keys = readJSON(`${__dirname}/keys.json`);
```
no info folder in this repo

**Pull request merged**